R/utils.R

```diff
@@ -286,3 +286,37 @@ check_results_structure <- function(data, name_data, cuc_cols) {
 
   return(data)
 }
+
+#' Assign value of flat multiplier
+#'
+#' Assign value of flat multiplier based on `asset_type`.
+#'
+#' @inheritParams run_stress_test
+#'
+#' @return A double holding value of the flat multiplier.
+assign_flat_multiplier <- function(asset_type) {
+  if (asset_type %in% c("loans", "bonds")) {
+    flat_multiplier <- 0.15
+  } else {
+    flat_multiplier <- 1.0
+  }
+  return(flat_multiplier)
+}
```

Comment:
Two simpler alternatives:

```r
if (asset_type %in% c("loans", "bonds") {
  return(0.15)
}
  
1.0
```
  
  Or 
  
  ```r
  ifelse(asset_type %in% c("loans", "bonds"), 0.15, 1.0)
  ```

(Inspired by https://speakerdeck.com/jennybc/code-smells-and-feels?slide=44)

Reply:
if jenny says so I will certainly do that, DONE